DayrSuryan/DSEthiop2.xml

```diff
@@ -359,6 +369,65 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                            <item xml:id="e3">
                               <locus target="#ir"/>
+
```

Comment:
This and the following extra descriptions are older versions not present like this anymore in the current state of this record in the master branch. If it is just a synchronizing accident, I can fix it easily without complications. But maybe it is not an accident? Did you chose the older version on purpose?

Reply:
No, sorry, it was an accident. It was a late time.

Reply:
No problem! I'll do it now, if you aren't working in the file at the same time